ZooManagement/ZooManagement_1.sql

```diff
@@ -0,0 +1,118 @@
+CREATE DATABASE ZooManagement;
+
+USE ZooManagement;
```

Comment:
I cannot add comments on the picture above so I'm adding them here:

1. A `Zoos` table would help because there might me many zoos that want to purchase your management app and you should be able to differentiate one to another.
2. The `Zoos` table could come up with an `Addresses` table as well so that you know where is each zoo located and with a relationship to the `Tickets` table so that you know which ticket is valid for a specific zoo.
3. A `ValidityTypes` table would be perfect to store the validity types of the tickets (considering there is only a few valid / possible options like one day, one week, one month entry) and build a relationship with the `Tickets` table.
4. The `Employee` table has one column named `EnclosureId` which suggests me that a relationship with the `Enclosures` table should be present but I see no relationship drawn on the diagram.
5. I like the fact that you used consistency and named all the tables as plural. It's great!
6. I think the relationship between `Tickets` and `Visitors` should be the other way around. A one to many but in reverse because a ticket can only belong to one visitor but a visitor can purchase many tickets (if he/she visits the zoo multiple times).
7. The many to many relationship between `Employee` and `Animals` would require an intermediate table `Employees_Addresses` to be able to be usable.